Source/WebCore/dom/Node.h

```diff
@@ -329,15 +329,6 @@ class Node : public EventTarget, public CanMakeCheckedPtr {
     WEBCORE_EXPORT bool isRootEditableElement() const;
     WEBCORE_EXPORT Element* rootEditableElement() const;
 
-    // Called by the parser when this element's close tag is reached,
-    // signaling that all child tags have been parsed and added.
-    // This is needed for <applet> and <object> elements, which can't lay themselves out
```

Comment:
Note this comment is outdated since applet and object don't override these functions anymore.